src/Kalambury.js

```diff
@@ -77,31 +79,40 @@ function UpdateDrawing(G, _ctx, lines) {
 
 function Forfeit(G, ctx) {
   G.points[ctx.currentPlayer] -= 1;
-  ctx.events.endTurn();
+  ctx.events.endTurn({ next: nextActivePlayer(G, ctx) });
```

Comment:
Can't we create custom turn order that will do this automatically? https://boardgame.io/documentation/#/turn-order?id=creating-a-custom-turn-order